livereveal/main.js

```diff
@@ -348,15 +351,18 @@ function buttonExit() {
 }
 
 function Remover() {
-  $('div#notebook_panel').css('background-color','#ffffff');
-  $('div#header-container').toggle();
+  $('div#site').css("height", "");  
```

Comment:
Yep. To be honest, I have only tested this in Chrome so far, but I'm pretty sure it works in other browsers too.